pymobiledevice3/services/crash_reports.py

```diff
@@ -127,28 +128,36 @@ def watch(self, name: str = None, raw: bool = False) -> Generator[str, None, Non
                 else:
                     yield crash_report
 
-    def get_new_sysdiagnose(self, out: str, erase: bool = True) -> None:
+    def get_new_sysdiagnose(self, out: str, erase: bool = True, *, timeout: Optional[float] = None) -> None:
```

Comment:
why the `*`? remove if it's not needed

Reply:
To force timeout as a keyword vs positional argument.  I find it:
1. Allows the addition of future positional arguments without breaking the method signature.
2. improves readability

I will remove the `*` if you do not agree.